### PR TITLE
Refactor leaflet examples. Add gmaps examples

### PR DESCRIPTION
**Pull request by Jesus89**
Related to: https://github.com/CartoDB/cartodb.js/issues/1934
---

### CODE REVIEW

File: examples/public/misc/edit-sql-cartocss-gmaps.html [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+        return document.querySelector(id).value
+      }
+
+      function reset(id) {
+        document.querySelector(id).style.background = 'white';
+      }
+
+      function error(id, cartoError) {
+        document.querySelector(id).style.background = '#E57373';
+        setTimeout(() => {
```
Comment (IagoLast) [inline examples/public/misc/edit-sql-cartocss-gmaps.html]: why this timeout?

Comment (Jesus89) [reply examples/public/misc/edit-sql-cartocss-gmaps.html]: This is to enable the color to be changed before the alert. But I accept new ideas.

File: examples/public/filters/bounding-box-gmaps.html [hunk 2]
```diff
@@ -0,0 +1,103 @@
...
+      function renderWidgetCategory(data) {
+        let content = '';
+        for (let category of data.categories) {
+          content += `<b>${category.name}</b><p><span>${parseInt(category.value)}</span> inhabitants</p>`;
+        }
+        document.getElementById('widgetCategory').innerHTML = content;
+      };
+
+      function renderWidgetFormula(data) {
+        let content = `<span>${data.result}</span> cities`;
```
Comment (IagoLast) [inline examples/public/filters/bounding-box-gmaps.html]: const

File: examples/public/filters/bounding-box-leaflet.html [hunk 3]
```diff
@@ -0,0 +1,96 @@
...
+        let content = '';
+        for (let category of data.categories) {
+          content += `<b>${category.name}</b>
+          <p><span>${parseInt(category.value)}</span> inhabitants</p>`;
+        }
+        document.getElementById('widgetCategory').innerHTML = content;
+      }
+
+      function renderWidgetFormula(data) {
+        let content = `<span>${data.result}</span> cities`;
```
Comment (IagoLast) [inline examples/public/filters/bounding-box-leaflet.html]: const

File: examples/public/filters/custom-bounding-box-leaflet.html [hunk 4]
```diff
@@ -0,0 +1,131 @@
...
+        let content = '';
+        for (let category of data.categories) {
+          content += `<b>${category.name}</b>
+          <p><span>${parseInt(category.value)}</span> inhabitants</p>`;
+        }
+        document.getElementById('widgetCategory').innerHTML = content;
+      }
+
+      function renderWidgetFormula(data) {
+        let content = `<span>${data.result}</span> cities`;
```
Comment (IagoLast) [inline examples/public/filters/custom-bounding-box-leaflet.html]: const

File: examples/public/misc/edit-sql-cartocss-gmaps.html (hunk 1 above)
Comment (IagoLast) [reply examples/public/misc/edit-sql-cartocss-gmaps.html]: 🔥

**Comment by IagoLast** [comment]
🏳️

**Pull request merged**